src/observers/link_click_observer.ts

```diff
@@ -33,8 +33,13 @@ export class LinkClickObserver {
   }
 
   clickBubbled = (event: MouseEvent) => {
+    // TODO: remove before any merge
+    console.log("event: ", event)
+    console.log("target is: ", event.target)
+    console.log("fixed target is: ", event.composedPath()[0])
```

Comment:
when click is triggered by a natural user click in the browser's GUI, this outputs:
<img width="1162" alt="Screen Shot 2021-05-23 at 15 03 27" src="https://user-images.githubusercontent.com/48771/119263187-53eb5e80-bbde-11eb-8c41-864eb174af8e.png">

when the lick is triggered using the WIP test case initially submitted in this PR, this outputs:
<img width="843" alt="Screen Shot 2021-05-23 at 15 02 22" src="https://user-images.githubusercontent.com/48771/119263225-6fef0000-bbde-11eb-87d9-f722893f76fa.png">

both in chrome and firefox